### PR TITLE
[Add] huetiful-js 

### LINKED ISSUE #1677: [huetiful-js] <https://github.com/prjctimg/huetiful>
### Site URL

https://prjctimg.github.io/huetiful

### Description of the above provided site

JavaScript library for simple color manipulations based on the Culori API.

### Category

Frontend

### Sub Category

"colors"

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by prjctimg**
Closes #1677

Added [huetiful-js](https://github.com/prjctimg/huetiful) to the Frontend category under the "colors" subcategory

### CODE REVIEW

File: database/frontend/colors.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
-[
+[  { 
+     "name": "huetiful-js", 
+     "description": "JavaScript library for simple color manipulations based on the Culori API. With over 50 utilities, it allows you to filter and sort color collections and query different properties of color as well as generate different palettes.",
```
Comment (CBID2) [inline database/frontend/colors.json:3]:
```suggestion
     "description": "This is a JavaScript library for simple color manipulations based on the Culori API. With over 50 utilities, Huetiful allows you to filter and sort color collections and query different properties of color as well as generate different palettes.", 
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @prjctimg. Your PR is off to a great start! I just have a suggestion on how you can improve it

**Review by CBID2 — DISMISSED** [review]
Well done @prjctimg! :) LGTM! :)

**Review by rupali-codes — COMMENTED** [review]
@prjctimg it looks perfect though, but could you please format the code?

**Comment by prjctimg** [comment]
Code has been formatted using default prettier config

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**